src/jit/lsra.cpp

```diff
@@ -6795,6 +6826,65 @@ void LinearScan::unassignPhysReg(RegRecord* regRec, RefPosition* spillRefPositio
     }
 }
 
+#ifdef _TARGET_ARM_
+//------------------------------------------------------------------------
+// unassignPhysReg: Unassign the given physical register that needs to consider currentInterval
```

Comment:
@CarolEidt I think it looks like to be better to use unassignDoublePhysReg() defined in https://github.com/dotnet/coreclr/pull/13145. So I would make another PR to use unassignDoublePhysReg().